login-fire-form.html

```diff
@@ -287,9 +292,9 @@
     _signUp: function(e) {
         this._cleanMessages();
         this.$$("#repeatPasswordInput").invalid=false;
-        e = this.$.emailInput.validate()
-        p = this.$.passwordInput.validate()
-        pRepeat = this.password == this.passwordAgain
+        e = this.$$("#emailInput").validate()
```

Comment:
Why do you prefer query selector over getting the element by ID? This feature will be removed from Polymer in the next version?